testsupport/cleanup/clean.go

```diff
@@ -181,7 +181,7 @@ func (c *cleanTask) verifyMurDeleted(isUserSignup bool, userSignup *toolchainv1a
 			if err := c.client.Get(context.TODO(), test.NamespacedName(userSignup.GetNamespace(), userSignup.Status.CompliantUsername), mur); err != nil {
 				// if MUR is not found then we are good
 				if errors.IsNotFound(err) {
-					c.t.Logf("the related MasterUserRecord: %s is deleted as well", userSignup.Status.CompliantUsername)
+					// c.t.Logf("the related MasterUserRecord: %s is deleted as well", userSignup.Status.CompliantUsername)
```

Comment:
I see that they might not provide huge value 99% of the times, but I do remember we had issues with cleaning up resources and those logs were useful that time. Should we keep them since they are just test logs ? 

Maybe in the test framework it's easier to swap the logger with something that supports log levels, and use those as debugging only.

Reply:
ok, I'll restore those changes

Reply:
restored in e22a8af